tests/wasmfs/wasmfs_create.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+// FIXME: Merge with other existing close and open tests.
+
+int main() {
+  // Test creating a new file and writing and reading from it.
+  int fd = open("/test", O_RDWR | O_CREAT);
+  const char* msg = "Test\n";
+  write(fd, msg, strlen(msg));
+  char buf[100];
```

Comment:
To zero-initialize the array and ensure we don't print junk:
```suggestion
  char buf[100] = {};
```